rust/lance-index/src/vector/graph.rs

```diff
@@ -172,6 +172,7 @@ pub(super) fn beam_search(
     start: &[u32],
     query: &[f32],
     k: usize,
+    bitset: Option<&roaring::bitmap::RoaringBitmap>,
```

Comment:
Could you describe what this bitset represents above in the "Parameters" section of the docs?

Looks like a set of "node_ids". How does one get the set of `node_ids` from the matching row ids (which are u64)?

Reply:
indices of row IDs are the node IDs for HNSW, and `PreFilter::filter_row_ids` returns the indices of row IDs to keep